<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>246</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>342</width>
    <height>246</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>342</width>
    <height>246</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Crypt v3</string>
  </property>
  <property name="windowIcon">
   <iconset theme="face-smile-big"/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>211</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans Bold Semi-Condensed'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;XORcrypt v3 by &lt;span style=&quot; color:#1a5fb4;&quot;&gt;E&lt;/span&gt;&lt;span style=&quot; color:#26a269;&quot;&gt;m&lt;/span&gt;&lt;span style=&quot; color:#e5a50a;&quot;&gt;p&lt;/span&gt;&lt;span style=&quot; color:#c64600;&quot;&gt;e&lt;/span&gt;&lt;span style=&quot; color:#a51d2d;&quot;&gt;r&lt;/span&gt;&lt;span style=&quot; color:#613583;&quot;&gt;s&lt;/span&gt;&lt;span style=&quot; color:#63452c;&quot;&gt;0&lt;/span&gt;&lt;span style=&quot; color:#9a9996;&quot;&gt;n&lt;/span&gt;&lt;span style=&quot; color:#241f31;&quot;&gt;_&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="threads">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>32</number>
    </property>
   </widget>
   <widget class="QLabel" name="thName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;thread&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xorStart">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>94</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>321</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Filepath</string>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>321</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="cryptProgress">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="hasNd">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ndpassword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>291</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Password2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>342</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
